spdx-utils/src/main/kotlin/model/SpdxRelationship.kt

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.spdx.model
+
+/**
+ * A relationship between two SPDX documents, packages or files.
+ */
+data class SpdxRelationship(
+    /**
+     * The source element of this directed relationship.
+     */
+    val spdxElementId: String,
+
+    /**
+     * The target element of this directed relationship.
+     */
+    val relatedSpdxElement: String,
+
+    /**
+     * The type of this relationship.
+     */
+    val relationshipType: Type
+) {
+    enum class Type {
+        /**
+         * Is to be used when (current) SPDXRef-DOCUMENT amends the SPDX information in SPDXRef-B.
```

Comment:
I think "Is to be used when" could be stripped of all comments here.
Replace "(current) SPDXRef-DOCUMENT" with "SPDXRef-A" as in comments below?

Reply:
Are you aware that for this enum I decided to use the original comments from the spec?

I'm not saying youre proposal does not make sense with this questions...

Reply:
No. Then fix it in the spec ;)

Reply:
Want me to change it here regardless of whether I updated the spec?